### PR TITLE
Simplified Exit logic in deploy.sh

### PR DESCRIPTION
**Pull request by nagamalli9999**
'set -e' will force bash to exit if there is a non zero exit code

**What kind of change does this PR introduce?**
This PR makes the deploy.sh more readable and easier to maintain

**What is the new behavior?**
There is no change in expected behavior

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [x] Ready to be merged 
- [x] Added myself to contributors table

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-echo "Untarring www files" && \
+rm -rf .deliver
+mkdir .deliver
+cp -R www .deliver
+echo "Tarring build files"
+tar -C .deliver -zcvf .deliver/code_sandbox_client.tar.gz www
+echo "Cleaning remote tar"
+ssh nginx@ssh.codesandbox.io "rm -f code_sandbox_client.tar.gz"
+echo "Moving tar to server"
+scp .deliver/code_sandbox_client.tar.gz nginx@ssh.codesandbox.io:./
```
Comment (lbogdan) [inline deploy.sh]: Do we need the whole `.deliver` thing? Why not just do
```
tar -zcvf code_sandbox_client.tar.gz www
[...]
scp code_sandbox_client.tar.gz nginx@ssh.codesandbox.io:
```

Comment (nagamalli9999) [reply deploy.sh]: This is a 0 logic change pull aimed at improving maintainability. I don't have enough context to make that change

Comment (lbogdan) [reply deploy.sh]: Sorry, the question was more for @CompuIves, i should have mentioned. 😄

Comment (CompuIves) [reply deploy.sh]: Hmm, yes. Yesterday we moved to the new deployment flow and this is kind of a leftover. We can just move to `www` directory now 😄 . Let's merge this one in first.

**Pull request merged**